app/main/views/services.py

```diff
@@ -246,6 +245,77 @@ def update_section_submission(service_id, section):
         )
 
 
+@main.route('/submission/g-cloud-7/create', methods=['GET'])
+@login_required
+@flask_featureflags.is_active_feature('CREATE_SERVICE_PAGE')
+def start_new_draft_service():
+    """
+    Page to kick off creation of a new (G7) service.
+    """
+    template_data = main.config['BASE_TEMPLATE_DATA']
+
+    lots = new_service_content.get_question('lot')
+    lots['type'] = 'radio'
```

Comment:
I'm pretty sure there was a rationale behind `radios` instead of `radio` in the SSP content (@quis can probably shed some light).  It might be better to stick with `radios` for this macro too for consistency?